contracts/controller/src/borrows_supplies.rs

```diff
@@ -131,7 +131,12 @@ impl Contract {
     ) -> bool {
         let existing_supplies = self.get_entity_by_token(Supply, account.clone(), token_address.clone());
 
-        return existing_supplies >= Balance::from(token_amount);
+        assert!(
```

Comment:
@dmytro-shash any pre-conditions check should be done as soon as possible (especially in blockchain where users pays for every executed operation.
also check `required!` macro which is more lightweight https://www.near-sdk.io/best-practices#use-assert-early

"withdraw is paused" is enough for error